static/js/components/PersonalTab_test.js

```diff
@@ -9,10 +9,11 @@ import getMuiTheme from 'material-ui/styles/getMuiTheme';
 import PersonalTab from './PersonalTab';
 import { PROGRAMS } from '../constants';
 import IntegrationTestHelper from '../util/integration_test_helper';
+import { USER_PROFILE_RESPONSE } from '../constants';
 
 describe("PersonalTab", () => {
   let helper;
-  let renderPersonalTab = (selectedProgram = null, props = {}) => {
+  let renderPersonalTab = (selectedProgram = null, props = {uneditedProfile: USER_PROFILE_RESPONSE}) => {
```

Comment:
Can you define this as an explicit prop below instead? Otherwise this would get overwritten when the user passes an argument for `props`